.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @seek-oss/indirect
```

Comment:
Reminder to change this

Reply:
Open to changing this in the PR if we can come to an agreement

Reply:
What about:

```suggestion
* @seek-oss/sig-backend-tooling @seek-oss/sku-maintainers 
```

(and adjusting the admin access as per ^)